Utilities/Utilities.csproj

```diff
@@ -54,25 +54,139 @@
     <Reference Include="MathNet.Numerics, Version=4.7.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\MathNet.Numerics.4.7.0\lib\net461\MathNet.Numerics.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Data.Sqlite, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
```

Comment:
Wow a huge load of MS stuff added to Utilities that I did not expect. Are we sure about all this?

Reply:
It was more than I expected too. It it weren't all MS stuff, I'd be much more reluctant.